Reflection/ReflectionClassRegistry.php

```diff
@@ -0,0 +1,71 @@
+<?php
+
+namespace Pim\Bundle\CustomEntityBundle\Reflection;
+
+use Doctrine\Common\Util\ClassUtils;
+use Symfony\Component\PropertyAccess\PropertyAccessorInterface;
+
+
+/**
+ * @author    Romain Monceau <romain@akeneo.com>
+ * @copyright 2017 Akeneo SAS (http://www.akeneo.com)
+ */
+class ReflectionClassRegistry
```

Comment:
Need help to find better naming and better namespace

Reply:
Same comment than previous class

Reply:
ClassMetadata